packages/local-signing-manager/src/lib/local-signing-manager.ts

```diff
@@ -108,18 +108,12 @@ export class LocalSigningManager implements SigningManager {
   }
 
   /**
-   * Create an instance of the Local Signing Manager and populates it with a newly generated Account
+   * Generate a new Polymesh account and return its mnemonic. This account can be used with the Local Signing Manager to sign transactions via the Polymesh SDK
+   *
+   * @note make sure to store the returned mnemonic somewhere safe since it will not be accessible by any means after being returned, and it gives full control over the account
    */
-  public static async generate(): Promise<{
-    mnemonic: string;
-    keyringPair: IKeyringPair;
-    signingManager: LocalSigningManager;
-  }> {
-    await cryptoWaitReady();
-
-    const signingManager = new LocalSigningManager([]);
-    const newAccount = signingManager._createAndAddAccount();
-    return { ...newAccount, signingManager };
+  public static generateAccount(): string {
```

Comment:
Isn't the `cryptoWaitReady` necessary for the `mnemonicGenerate` call?

Reply:
It works even without it

Reply:
I looked at it and if we ensure ready, it uses `bip39Generate(numWords)` which is part of `@polkadot/wasm-crypto` else it uses `generateMnemonic(numWords)` which is within `@polkadot/util-crypto` itself (bip39.js)